students/jinsungpark/users/views.py

```diff
@@ -43,14 +45,20 @@ def post(self, request):
 class LogInView(View):
     def post(self, request):
         try:
-            data      = json.loads(request.body)
-            email     = data["email"]
-            password  = data["password"]
+            data  = json.loads(request.body)         
+            user  = User.objects.get(email=data["email"])
+            if bcrypt.checkpw(data["password"].encode('utf-8'), user.password.encode('utf-8')):
+                access_token = jwt.encode({'id' : user.id}, SECRET_KEY, algorithm=ALGORITHM)
 
-            if not User.objects.filter(email=email, password=password).exists():
-                return JsonResponse({"message":"INVALID_USER"}, status=401)
-
-            return JsonResponse({"message" : "SUCCESS"}, status=200)
+                return JsonResponse({"message" : access_token}, status=200)    
+                
+            return JsonResponse({"message" : "UNAUTHORIZATION"}, status=401)
```

Comment:
`unathorized`는 비밀번호 오류 메세지라고 보기엔 너무 포괄적입니다.
```suggestion
            return JsonResponse({"message" : "invalid_password"}, status=401)
```

Reply:
수정완료했습니다